src/Tests/engine/EntityCheck.cpp

```diff
@@ -1,13 +1,11 @@
 #include <catch.hpp>
 #include <Engine/Entity/Entity.hpp>
 
-TEST_CASE("", "[entity component]")
-{
+TEST_CASE("", "[entity component]") {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The function 'TEST_CASE' is never used. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)